vllm/model_executor/sampling_metadata.py

```diff
@@ -466,22 +465,30 @@ def from_lists(cls, temperatures: List[float], top_ps: List[float],
         do_penalties = prompt_tokens or output_tokens
 
         if do_penalties:
-            prompt_max_len = max([len(tokens) for tokens in prompt_tokens],
+            prompt_max_len = max((len(tokens) for tokens in prompt_tokens),
```

Comment:
Using a generator expression can avoid iterating through the tokens again when calling `max`.